src/domain/challenge/ecoverse/ecoverse.resolver.mutations.ts

```diff
@@ -119,18 +119,20 @@ export class EcoverseResolverMutations {
   @Profiling.api
   async createChallenge(
     @CurrentUser() agentInfo: AgentInfo,
-    @Args('challengeData') challengeData: CreateChallengeInput
+    @Args('challengeData') challengeData: CreateChallengeInEcoverseInput
```

Comment:
All the rest of the mutations are xxxOnxxx, and here is xxxInxxx, it is not consistent with the rest of the API